src/core/contracts/identity.test.ts

```diff
@@ -656,19 +664,22 @@ describe("identity", () => {
           identityAddress: "0x71C95911E9a5D330f4D621842EC243EE1343292e",
           delegate: "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
           blockNumber: 330,
+          transactionHash: expect.any(String),
         },
         {
           name: "DSNPAddDelegate",
           identityAddress: "0x8464135c8F25Da09e49BC8782676a84730C318bC",
           delegate: "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
           blockNumber: 332,
+          transactionHash: expect.any(String),
         },
         {
           name: "DSNPRemoveDelegate",
           identityAddress: "0x8464135c8F25Da09e49BC8782676a84730C318bC",
           delegate: "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
           blockNumber: 333,
           endBlock: 400,
+          transactionHash: expect.any(String),
```

Comment:
not sure if it makes sense here, but you could do a regex match here to make sure it's a hash

```
transactionHash: expect.match(/^0x[0-9a-f]{64}/i)
```

Reply:
I like it but not sure if it adds much value for this test.